sekoia_automation/connector/__init__.py

```diff
@@ -29,10 +41,44 @@ class DefaultConnectorConfiguration(BaseModel):
 
 
 class Connector(Trigger, ABC):
-    configuration: DefaultConnectorConfiguration
+    CONNECTOR_CONFIGURATION_FILE_NAME = "connector-configuration"
+    default_configuration: DefaultConnectorConfiguration
```

Comment:
This is dangerous, all the connectors using the SDK rely on `configuration`, not ` default_configuration`. This change will break them. I think you can keep this attribute that is use for typing and have the properties

```suggestion
    configuration: DefaultConnectorConfiguration
```